### ISSUE TITLE
Offload data from DLM to an external device

### ISSUE THREAD

**Issue by caljay98** [gopher-motorsports/data-logging-module#10]
Create a way to move the data stored on the DLM persistent storage onto the tracktop. Creating a http server on the DLM and accessing it through Ethernet would be great

**Comment by hocally** [comment]
@caljay98 HTTP server sounds great. @BTyrrell01, did you ever have any more luck on LWIP?

**Comment by BTyrrell01** [comment]
Nope, it would always time out and crash. The code is on Github though I believe if not I can make a repo for it

**Comment by caljay98** [comment]
For REV1 we should choose the method of sending data

**Comment by caljay98** [comment]
A test program has been written, so Ethernet is a GO

**Comment by BTyrrell01** [comment]
HE HAS DONE IT

**Comment by hocally** [comment]
@caljay98 C god confirmed
